index.html

```diff
@@ -1,114 +1,125 @@
 <!DOCTYPE html>
 <html lang="en">
-    <head>
-        <title></title>
-        <meta charset="UTF-8">
-        <meta name="viewport" content="width=device-width, initial-scale=1">
-        <meta name="google-signin-scope" content="profile email">
-        <meta name="google-signin-client_id" content="89672675826-ut68sal4k0dljj7ocn098slba69g053s.apps.googleusercontent.com">    
-        <link href="https://fonts.googleapis.com/css?family=Monoton|Nunito+Sans" rel="stylesheet">
-        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/uikit/3.0.0-rc.20/css/uikit.min.css"/>
-        <link rel="stylesheet" href="node_modules/frow/dist/frow.min.css">
-        <link rel="stylesheet" href="assets/css/styles.css">
-    </head>
-    <body>
 
-        <header>Tempestatibus</header>
-        <hr class="uk-divider-icon">
-        
-        <div id="weather-hero">
-            <div id="weather-main">
-                <div class="uk-card uk-card-default uk-card-body uk-width-1-2@m weather-card">
-                    <div class="card-header">
-                        CURRENT WEATHER
-                    </div>
-                    <div id="location-current">Houston, TX
-                        <img src="assets/climacons-master/Cloud-Hail-Moon.svg">
-                        34 F
+<head>
+    <title></title>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta name="google-signin-scope" content="profile email">
+    <meta name="google-signin-client_id" content="89672675826-ut68sal4k0dljj7ocn098slba69g053s.apps.googleusercontent.com">
+    <link href="https://fonts.googleapis.com/css?family=Monoton|Nunito+Sans" rel="stylesheet">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/uikit/3.0.0-rc.20/css/uikit.min.css" />
+    <link rel="stylesheet" href="node_modules/frow/dist/frow.min.css">
+    <link rel="stylesheet" href="assets/css/styles.css">
+    <script src="https://code.jquery.com/jquery-3.3.1.min.js" integrity="sha256-FgpCb/KJQlLNfOu91ta32o/NMZxltwRo8QtmkMRdAu8="
+        crossorigin="anonymous"></script>
+    <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyDa2h9qlaLd0YfUtXstdOGfGtwojoTC50M&libraries=places,geometry"></script>
+    <script src='https://cdnjs.cloudflare.com/ajax/libs/skycons/1396634940/skycons.min.js'></script>
 
```

Comment:
Please move lines 14-17 down to line 115, so that we are keeping all of our JavaScript links in the same place.